src/path-symex/path_symex.cpp

```diff
@@ -290,6 +297,101 @@ void path_symext::symex_malloc(
   assign(state, lhs, rhs);
 }
 
+
+/*******************************************************************\
+
+Function: get_symbol
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+irep_idt get_symbol(const exprt &src, path_symex_statet &state)
```

Comment:
Make it static; get_symbol is also really generic, what does it really do? The state parameter should be a const reference